--- audacity-2.1.2.orig/src/FileNames.cpp
+++ audacity-2.1.2/src/FileNames.cpp
@@ -113,7 +113,8 @@ wxString FileNames::DataDir()
          // Use OS-provided user data dir folder
          wxString dataDir;
 #if defined( __WXGTK__ )
-         dataDir = wxStandardPaths::Get().GetUserDataDir() + wxT("-data");
+         //dataDir = wxStandardPaths::Get().GetUserDataDir() + wxT("-data");
+         dataDir = wxStandardPaths::Get().GetUserConfigDir() + "/.config/audacity-data";
 #else
          dataDir = wxStandardPaths::Get().GetUserDataDir();
 #endif
